src/AuthContext.tsx

```diff
@@ -9,7 +9,7 @@ import {
   epochTimeIsPast,
 } from './authentication'
 import useLocalStorage from './Hooks'
-import { IAuthContext, IAuthProvider, TInternalConfig, TTokenData, TTokenResponse } from './Types'
+import { IAuthContext, IAuthProvider, TInternalConfig, FormEncoding, TTokenData, TTokenResponse } from './Types'
```

Comment:
```suggestion
import { IAuthContext, IAuthProvider, TInternalConfig, TTokenData, TTokenResponse } from './Types'
```
This was removed with your latest commit, and now caused an import error